base/GoldenSun.ts

```diff
@@ -15,7 +15,7 @@ import {ShopMenu} from "./main_menus/ShopMenu";
 import {InnMenu} from "./main_menus/InnMenu";
 import {ControlManager} from "./utils/ControlManager";
 import {CursorManager} from "./utils/CursorManager";
-import {Gamepad} from "./Gamepad";
+import {Gamepad as XGamepad, Button, CButton} from "./XGamepad";
```

Comment:
Do we really need `Button` and `CButton`? Can't we have just one variable?

Reply:
I want to separate the real GBA button with the other possible controller/engine buttons.